classes/Infrastructure/Auth/SymfonySentinelSession.php

```diff
@@ -13,13 +13,13 @@
 
 namespace OpenCFP\Infrastructure\Auth;
 
-use Cartalyst\Sentinel\Sessions\SessionInterface as SentinelSessionInterface;
-use Symfony\Component\HttpFoundation\Session\SessionInterface as SymfonySessionInterface;
+use Cartalyst\Sentinel\Sessions;
+use Symfony\Component\HttpFoundation;
```

Comment:
In this file, we're adapting two different `SessionInterfaces`s. The aliases did improve readability here. 😕